### PR TITLE
Run uniformly on docker-compose vs. docker compose plugin.

### PR DESCRIPTION
**Pull request by fhaftmann**
This makes the whole matter more robust wrt. details of the hosting environment.

E. g. at least the official docker repositories for Ubuntu 22 (https://download.docker.com/linux/ubuntu jammy/stable) do _not_ provide a dedicated _docker-compose_ executable but a docker compose plugin.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,7 @@ KUBECONFIG := $(shell pwd)/.kubeconfig
 
 # Default values
 DOCKER_COMPOSE_OVERRIDE=
+DOCKER_COMPOSE=$(shell if command -v docker-compose > /dev/null; then echo 'docker-compose'; else echo 'docker compose'; fi)
```
Comment (Gerrit91) [inline Makefile:12]: Can you explain this condition? Both branches return `docker-compose` in the end?

**Review by Gerrit91 — COMMENTED** [review]
Hey, thanks for contribution.

Running the actions workflow is hard for outside collaborators because the org secrets are not sent to workflow runs of repository forks for security reasons.

File: Makefile (hunk 1 above)
Comment (fhaftmann) [reply Makefile:12]: Mind the dash. You either geht `docker-compose` (docker compose stand-alone exeuctable) or `docker compose` (docker compose plugin).

Comment (Gerrit91) [reply Makefile:12]: Ah. Now I can see the dash. Thanks for explanation. 🙈

**Review by Gerrit91 — APPROVED** [review]

**Comment by Gerrit91** [comment]
Do you want to add the containerlab binary as well? (as proposed in #121)

**Comment by fhaftmann** [comment]
> Do you want to add the containerlab binary as well? (as proposed in #121)

That is the plan. Indeed, when preparing this PR, I came to the conclusion that the proposal in #121 is suboptimal.

**Pull request merged**